modules/5_install/templates/install_vars.yaml

```diff
@@ -11,3 +11,25 @@ log_level: ${log_level}
 release_image_override: '${release_image_override}'
 
 rhcos_kernel_options: [%{ for opt in rhcos_kernel_options ~}"${opt}"%{ endfor ~}]
+
+sysctl_tuned_options: ${sysctl_tuned_options}
+%{ if sysctl_tuned_options ~}
+
+sysctl_options:
+%{ for opt in sysctl_options ~}
+  -  ${opt}
+%{ endfor ~}
+
+match_array:
+%{ for opt in match_array_info ~}
+%{ if opt.label != "" ~}
```

Comment:
You could use nested loop here.

Reply:
I am still not convinced with so many conditions.. nested loop should work here since you are anyway limiting the keys in TF code.